### ISSUE TITLE
Bandwidth set in SDP for non-unifiedPlan (multicast) seems wrong

### ISSUE THREAD

**Issue by ElielCohen** [aws/amazon-chime-sdk-js#1464]
### What happened and what did you expect to happen?

Hi Everyone,

To my understanding, we are wrongly setting the AS value in SDP for non unified plan : 

Here is code excerpt from the relevant file :

**src/sdp/DefaultSDP.ts**

200 | dstLines.push(`b=TIAS:${maxBitrateKbps * 1000}`);
201 | } else {
202 | dstLines.push(`b=AS:${maxBitrateKbps}`);

It seems that we should as well, for non unifiedPlan, have AS set to maxBitRateKbps*1000, since this value is in bytes.

Thanks,

Eliel Cohen
WebRTC Tech Lead
TytoCare

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

It's a code implementation issue, simply look at the code source of the project.

### Amazon Chime SDK for JavaScript version

Does exist in all version of the Chime JS SDK 

### What browsers are you seeing the problem on?

N/A

### Browser version

N/A

### Browser console logs

N/A

**Comment by hensmi-amazon** [comment]
Good catch, I think the code was originally assuming that only Firefox would be using unified plan.  We will take a look.

**Comment by ElielCohen** [comment]
Any update ... ?
